automated/linux/cyclictest/parse_results.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# SPDX-License-Identifier: MIT
```

Comment:
I don't think this is enough to make it 'real open source'. Could you add full license header and copyright?

Reply:
I've added the header. BTW, I am not insisting on MIT. If you prefer GPL let me know.

Reply:
Any open source license works for me. I just need to make sure this is a 'proper' open source project so we're not violating US entity list restrictions. Open source is exempt from that.